torchvision/io/__init__.py

```diff
@@ -123,19 +123,20 @@ def __next__(self):
     def __iter__(self):
         return self
 
-    def seek(self, time_s: float):
+    def seek(self, time_s: float, keyframes_only=False):
```

Comment:
Should we have type annotation here as well?

Reply:
yup